plugins/metas.ts

```diff
@@ -67,62 +72,84 @@ export default function (userOptions?: Partial<Options>) {
         return;
       }
 
+      const getMetaValue = <T extends keyof MetaData>(key: T) => {
+        if (key in metas) {
```

Comment:
`_date.metas.title` get merged into `page.metas.title` before this plugin's code execute. So this `metas.title` has value even if `page.metas.title` is not set.

Reply:
If you want to use fallbacks, you shouldn't add your own default value. For example, `_date.metas.title` shouldn't be defined.

Another option is remove the `metas.title` in the page, so the fallback value is used:

```yml
---
title: Hello world
metas:
  title: # This is empty, so the title fallback should be used
---
```

Reply:
From my point of view the second option is more acceptable.

What is the value of the empty fallback case, is it `undefined` or empty string. Sorry i am on mobile, cant check now.

Reply:
In this page, `metas.title` is `null` (no matter if there's a `_data.metas.title` with other value, because it's overriden). The plugin must use the `title` variable.

Reply:
Ok, I will update later.